config/fields/date.php

```diff
@@ -126,51 +105,44 @@
 
             return $this->time['step'];
```

Comment:
I think we have an issue here (also had before) that we never teat the sub options of `time` (e.g. `step`) as we prepare them on the top-level

Reply:
Sorry, I'm lost here. What do you mean exactly :)

Reply:
In this computed property we either use the main step option or if a time option is defined, we use the step sub-option of the time option.

But while we would run the main step option through the Date class method, we just use the step sub-option as is.

Reply:
Ah, I think I got it.